minitests/iostandard/features/README.md

```diff
@@ -0,0 +1,41 @@
+# IOSTANDARD feature correlation minitest
+
+This test checks which fasm features present in the db are set for given IO settings (IOSTANDARD + DRIVE + SLEW) and for which IOB type (IBUF, OBUF, IOBUF). It also checks what features are set on unused IOBs of a given bank in which active IOBs are instantiated. To avoid conflicts, a single iosettings combination is used within an IO bank.
```

Comment:
Should add differential signalling examples (OBUFDS, OBUFTDS, IOBUFDS, etc)?

Reply:
It does. I just forgot to update the README. Now its fixed.